src/scss/mega/articles.scss

```diff
@@ -1,18 +0,0 @@
-@if $o-header-is-silent == false {
```

Comment:
The original plan (I'm not aware of the status of it) was to create several more modules in addition to this.